.github/workflows/docker.yml

```diff
@@ -32,10 +32,10 @@ jobs:
             VERSION_TAG=v${VERSION_TAG}-sha-${GITHUB_SHA::8}-SNAPSHOT
           fi
           echo ::set-output name=versionTag::${VERSION_TAG}
-      - name: Set up JDK 1.8
```

Comment:
While we're here, the JDK setup syntax has changed (11 instead of 1.11, includes distribution), along with a newer v2 version for the action:

```yml
      - name: Set up JDK 11
        uses: actions/setup-java@v2
        with:
          java-version: '11'
          distribution: 'adopt'
```

I'd suggest we go ahead and update to `setup-java@v2` and the new syntax.

See https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle

@lionel-nj A separate issue - I just noticed that we're mixing `actions/checkout@v1` and `actions/checkout@v2` in these files too - it should be `actions/checkout@v2` thoughout.

Reply:
Done.